README.md

```diff
@@ -10,15 +10,58 @@ None
 
 | variable | description | default |
 |----------|-------------|---------|
-
+| `jenkins_slave_user` | Jenkins user | `{{ __jenkins_slave_user }}` |
+| `jenkins_slave_group` | Jenkins group | `{{ __jenkins_slave_group }}` |
+| `jenkins_slave_home` | Jenkins home | `{{ __jenkins_slave_home }}` |
+| `jenkins_slave_package` | package name | `{{ __jenkins_slave_package }}` |
```

Comment:
Example playbook should be a working example, and the example in the README does not work. to ensure the example always works, just update the example with `tests/serverspec/default.yml`, which is supposed to work.